README.md

```diff
@@ -69,6 +70,40 @@ Update BIOS firmware
     New BIOS Firmware Version:
             ATF--v2.2(release):4.7.0-25-g5569834, UEFI--4.7.0-42-g13081ae
 
+### Update FRU data
+
+The following OEM fields can be modified by the user:
+
+- Product Manufacturer
+- Product Serial Number
+- Product Part Number
+- Product Version
+- Product Extra
+- Product Manufacture Date (format is "DD/MM/YYYY HH:MM:SS")
+- Product Asset Tag
+- Product GUID (Chassis Extra in ipmitool)
+
+All OEM fields must be set in the command. If a specified FRU field is left empty, the value for that field will be ignored but not overridden.
```

Comment:
If a specified FRU field is left empty => let's give example.

Reply:
Done